### PR TITLE
[xodr] SmallTownRoads.xodr

### PR DESCRIPTION
**Pull request by stonier**
# 🎉 New feature

A map from the JPN mining project that requires geometric spirals support, see #260.

## Test it

No testing yet. Think of it as a test-driven stub awaiting support from #260.

### CODE REVIEW

File: resources/SmallTownRoads.xodr
Comment (francocipollone) [inline resources/SmallTownRoads.xodr:1]: We are missing to install it in CMake world. Check the root `CMakeLists` file. (We don't have a glob there.)

**Review by francocipollone — CHANGES_REQUESTED** [review]
Thanks! one minor comment to go

File: resources/SmallTownRoads.xodr
Comment (stonier) [reply resources/SmallTownRoads.xodr:1]: Got it, added.

**Review by francocipollone — APPROVED** [review]
LGTM!

**Pull request merged**